### ISSUE TITLE
Signed int `T::MAX` constant incorrectly reported as private

### ISSUE THREAD

**Issue by Silvea12** [intellij-rust/intellij-rust#10499]
## Environment

* **IntelliJ Rust plugin version:** 0.4.194.5382-231
* **Rust toolchain version:** 1.69.0
* **IDE name and version:** CLion 2023.1.3
* **Operating system:** Linux AMD64

## Problem description
Signed ints have the `MAX` constant (maybe others too) incorrectly reported as private under [E0603](https://doc.rust-lang.org/error-index.html#E0603).
This does not affect unsigned ints, nor floats.

## Steps to reproduce
```rs
fn main() {
    let a = i8::MAX;
}
```

![image](https://github.com/intellij-rust/intellij-rust/assets/3120639/346a86fb-19db-4ca1-8a71-08f197395e7a)

**Comment by anas-elgarhy** [comment]
I noticed this issue also and I think it's soo weird because it works with some types

**Comment by Maksim20023** [comment]
Hi! I'm closing this ticket because the issue is fixed (please see a fix [Pull 10688](https://github.com/intellij-rust/intellij-rust/pull/10688)). This fix is included in the Rust plugin 0.4.199.5414 version. Please make sure that your installed plugin is [up-to-date](https://plugins.jetbrains.com/plugin/8182-rust/docs/rust-quick-start.html#update).  If this issue still reproduces for you, please reopen the ticket.
